### PR TITLE
feat: CSPM-1813 Ensure AWS Compute Optimizer is enabled

### PR DESCRIPTION
**Pull request by shreeti-shrestha**
Jira Link:

[CSPM-1813](https://plerion.atlassian.net/browse/CSPM-1813)

Summary/Motivation:

Added ComputeOptimizer and ApplicationAutoScaling as services to the Plerion asset counter to ensure their instances are counted in total assets

Testing Strategy:

Locally tested by running the counting script aws.js manually
Dependencies:
@aws-sdk/client-compute-optimizer

[CSPM-1813]: https://plerion.atlassian.net/browse/CSPM-1813?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

**Review by plerionedmund — COMMENTED** [review]
what about other scaling policy?

**Comment by shreeti-shrestha** [comment]
Added AWS::AutoScaling::ScalingPolicy as well. Thanks 🙏

File: AWS/aws.js [hunk 1]
```diff
@@ -117,16 +123,20 @@ const NEPTUNE_DB_CLUSTER_PARAMETER_GROUP =
 const ECS_TASK_DEFINITION = "AWS::ECS::TaskDefinition";
 const WAF_RULE_GROUP = "AWS::WAFv2::RuleGroup";
 const EKS_NODE_GROUP = "AWS::EKS::NodeGroup";
+const COMPUTE_OPTIMIZER_ENROLLMENT_STATUS =
```
Comment (plerionedmund) [inline AWS/aws.js:126]: dont forget to add the new resource type here too

Comment (shreeti-shrestha) [reply AWS/aws.js:126]: AWS::AutoScaling::ScalingPolicy is added directly to aws-dependencies.json actually

File: AWS/aws.js [hunk 2]
```diff
@@ -883,6 +891,38 @@ const queryNeptuneClusterParameterGroups = async (
...
+  const resources = [];
+  const command = new GetEnrollmentStatusCommand({});
+  const response = await client.send(command);
+  resources.push({ ...(response || {}) });
+  updateResourceTypeCounter(serviceName, resourceType, resources.length);
+  total += resources.length;
+  AWS_MAPPING.total += total;
+};
+
+const queryScalingPolicies = async (serviceName, resourceType, region) => {
```
Comment (plerionrikee) [inline AWS/aws.js:910]: Do we need this code block as you're importing ScalingPolices from aws-dependencies.json

Comment (shreeti-shrestha) [reply AWS/aws.js:910]: This one is for `AWS::ApplicationAutoScaling::ScalingPolicy` and the other one is `AWS::AutoScaling::ScalingPolicy`

**Review by plerionrikee — APPROVED** [review]
LGTM 👍

**Pull request merged**